model/configuration.py

```diff
@@ -182,6 +182,7 @@ class ExternalIntegration(Base, HasFullTableCache):
     FEEDBOOKS = DataSourceConstants.FEEDBOOKS
     LCP = DataSourceConstants.LCP
     MANUAL = DataSourceConstants.MANUAL
+    PROQUEST = DataSourceConstants.PROQUEST
```

Comment:
You also need to add this to `DataSource.well_known_sources`. This isn't as important as it used to be but it tells the system two important things:

1. ProQuest offers actual books you can read, not just metadata (`offers_licenses`)
2. ProQuest books are always identified by their ProQuest Doc ID (`primary_identifier_type`)

`offers_metadata_lookup` should be false. `refresh_rate` is no longer used so just set it to None.

Reply:
Added `ProQuest` to `DataSource.well_known_sources`